core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/configuration/script/ScriptConfiguration.java

```diff
@@ -384,4 +438,10 @@ private String getInsertStatement(Script script) {
         }
     }
 
+    public String getCurrentTimeStampAsString(){
+        SimpleDateFormat sdf1 = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+        Timestamp timestamp = new Timestamp(System.currentTimeMillis());
+        return sdf1.format(timestamp);
+    }
```

Comment:
have a look at the SQLTools of the iesi-core project and replace this logic with the logic implemented there